app/src/main/java/mozilla/lockbox/support/Constant.kt

```diff
@@ -25,6 +25,14 @@ object Constant {
         val scopes = arrayOf(profileScope, lockboxScope, oldSyncScope)
     }
 
+    object Faq {
+        const val uri = "https://mozilla-lockbox.github.io/lockbox-android/"
```

Comment:
This should point to "https://lockbox.firefox.com/faq.html".

CC @mozilla-lockbox/product and @mozilla-lockbox/ux to confirm the link.